src/renderers/webgl/index.ts

```diff
@@ -99,14 +98,6 @@ export const defaultOptions = {
   dragInertia: 0.88
 }
 
-// TODO - make configurable
-export const MIN_LABEL_ZOOM = 0.25
```

Comment:
importing these in other files started a circle dependency so I moved them to `./utils`